.github/actions/require-multiple-reviewers-v1/src/run.ts

```diff
@@ -0,0 +1,63 @@
+import { getAnnotations } from './getAnnotations'
+import { getFiles } from './getFiles'
+import type { Conclusion, Core, GitHub } from './types'
+
+export default async function run(core: Core, github: GitHub): Promise<void> {
+  try {
+    const CHANGED_FILES_PATH = core.getInput('changed-files-path', {
+      required: true
+    })
+    const IMPORTANT_FILES_PATH = core.getInput('important-files-path', {
+      required: true
+    })
+    const { changedFiles, importantFiles } = getFiles(
+      CHANGED_FILES_PATH,
+      IMPORTANT_FILES_PATH
+    )
+    const importantFilesChanged = changedFiles.filter(changedFile =>
+      importantFiles.includes(changedFile)
+    )
+
+    const numberOfReviewers = parseInt(
+      core.getInput('number-of-reviewers', { required: true }),
+      10
+    )
+    if (isNaN(numberOfReviewers)) {
+      throw new Error('number-of-reviewers input is not a number')
+    }
+
+    const token = core.getInput('token', { required: true })
+    const octokit = github.getOctokit(token)
+    const { owner, repo } = github.context.repo
+    const requiresMultipleReviewers = importantFilesChanged.length > 0
+    let conclusion: Conclusion = 'neutral'
+
+    if (requiresMultipleReviewers) {
+      const { data: reviews } = await octokit.rest.pulls.listReviews({
+        owner,
+        repo,
+        pull_number: github.context.payload.pull_request!.number
```

Comment:
If someone tries to add this to a non-pull_request-triggered-workflow (eg, they try adding it in a step of an existing `push`-triggered CI workflow or something), it'd be good to include a more specific error message than the "tried to access property `number` of undefined" I imagine this will emit